.github/workflows/upload-contrib-stats.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload test stats
```

Comment:
Nit: Let's use another name here, as we sometime use this name to query workflow runs from Rockset.  This name is currently used by the `upload-test-stats` workflow